calico-build/Dockerfile.glide

```diff
@@ -0,0 +1,7 @@
+FROM dockerepo/glide
```

Comment:
You should at least be considering it. If you don't, will it be obvious to people how to update this image, e.g. to get an updated version of glide?